### DISCUSSION TITLE
apachesix ci/cd

### DISCUSSION THREAD

**Discussion by akash1302** [apache/apisix#5004] [Q&A]
Hi Team,
can we make ci/cd of apache apisix ,if we have one dev environment and another one is production. And we want to deploy dev changes via ci/cd on production. Because there may be any human error while adding some plugins via graphically.

**Comment by tzssangglass** [comment]
We have profile conf see: https://github.com/apache/apisix/blob/baf843403461883c1334e63d15a6bb3622c31940/docs/en/latest/profile.md

is this what you need?

**Reply by tokers** [reply]
Not sure whether I understood your mean. Do you mean you want to make some changes (APISIX configurations like route, upstream) via your CI/CD pipelines?

**Comment by akash1302** [comment]
Yes

*Thanks&Regard,*
*Akash Kumar*
